### ISSUE TITLE
Support Terraform 0.11 module sources

### ISSUE THREAD

**Issue by brainsik** [wata727/tflint#167]
Terraform 0.11 introduced support for module installation from Terraform Registry. As a consequence, module source strings using relative paths must be expressed explicitly by beginning the string with `./` or `../`.  See [Relative Paths in Module source](https://www.terraform.io/upgrade-guides/0-11.html#relative-paths-in-module-source).

`tflint` is using the old interpretation of module `source`:

```
$ tflint --version
TFLint version 0.5.4
$ tflint 
ERROR: module `terraform-aws-modules/vpc/aws` not found. Did you run `terraform get`?
```

**Comment by davewongillies** [comment]
I'm hitting on this issue too when using the Terraform Enterprise private module registry:

```
$ tflint
ERROR: module `app.terraform.io/my-org/my-provider/my-module` not found. Did you run `terraform get`?
```

**Comment by wata727** [comment]
Currently, TFLint's module resolver is incomplete. Until now applied a partial patch from Terraform's implementation, but it seems that there was a big change in 0.11.

In the future, I plan to solve this problem by using Terraform's module resolver. However, It is necessary to wait for new Terraform version.

**Comment by davewongillies** [comment]
For those looking for a workaround, I'm making use of `tflint`'s `--ignore-module` option, which can also be set in `.tflint.hcl`

`.tflint.hcl:`
```hcl
config {
  ignore_module = {
    "app.terraform.io/my-org/my-provider/my-module" = true
  }
```

**Comment by wata727** [comment]
Hi there, I just released TFLint v0.8.0 that supports v0.11 and v0.12 modules. Please try the latest version again. If you still have problems, please open a new issue instead of reopening.
